src/current/cockroachcloud/cluster-management.md

```diff
@@ -165,9 +165,9 @@ When you remove a region from a [multi-region](plan-your-cluster.html#multi-regi
   
 ## Set an upgrade window
 
-From your cluster's [**Overview** page](cluster-overview-page.html), you can view and manage the patch upgrade window for your cluster. To help keep your clusters updated while minimizing disruption and downtime, set a window of time when your cluster is experiencing the lowest traffic.
+From your cluster's [**Overview** page](cluster-overview-page.html), you can view and manage the [patch upgrade](upgrade-policy.html#patch-version-upgrades) window for your cluster. During the window, your cluster may experience restarts, degraded performance, and downtime for single node clusters. To help keep your clusters updated while minimizing disruptions, set a window of time when your cluster is experiencing the lowest traffic. Note that upgrades may not always be completed by the end of the window, and other kinds of cluster maintenance may occur outside the window.
```

Comment:
```suggestion
From your cluster's [**Overview** page](cluster-overview-page.html), you can view and manage the [patch upgrade](upgrade-policy.html#patch-version-upgrades) window for your cluster. During the window, your cluster may experience restarts, degraded performance, and downtime for single-node clusters. To help keep your clusters updated while minimizing disruptions, set a window of time when your cluster is experiencing the lowest traffic. Note that upgrades may not always be completed by the end of the window, and other kinds of cluster maintenance may occur outside the window.
```
I think you'd also get downtime if you had a two-node cluster.  I wonder if it's worth just saying "reduced availability during the maintenance window"?

Reply:
i think this would be true, but we don't allow 2-node clusters on dedicated